probe/appclient/app_client.go

```diff
@@ -75,7 +75,7 @@ func NewAppClient(pc ProbeConfig, hostname, target string, control xfer.ControlH
 			TLSClientConfig: httpTransport.TLSClientConfig,
 		},
 		conns:   map[string]xfer.Websocket{},
-		readers: make(chan io.Reader),
```

Comment:
Had to pick a number, and two allows for a pod deleted followed by a pod created event to be queued up.